app/presenters/guest_presenter.rb

```diff
@@ -18,4 +18,8 @@ def win_header_partial
   def nav_drawer_submenu_partial
     "components/guest_nav_drawer_submenu"
   end
+
+  def welcome_message_partial
```

Comment:
Since these methods are not being called in one particular view but are "view-related" components, I think that putting them in a presenter makes sense. Basically, right now we have a 1:1 mapping of view models and controller methods. Presenters are decoration around that so they're a bit different. We can def discuss more, interested in why you think this should be a view model!